Index: SweetMangementSystem/src/test/java/TestUtils.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/SweetMangementSystem/src/test/java/TestUtils.java b/SweetMangementSystem/src/test/java/TestUtils.java
new file mode 100644
--- /dev/null	(date 1723147397870)
+++ b/SweetMangementSystem/src/test/java/TestUtils.java	(date 1723147397870)
@@ -0,0 +1,21 @@
+import java.awt.Component;
+import java.awt.Container;
+
+public class TestUtils {
+    public static Component getChildNamed(Component parent, String name) {
+        if (name.equals(parent.getName())) {
+            return parent;
+        }
+
+        if (parent instanceof Container) {
+            Component[] children = ((Container) parent).getComponents();
+            for (Component child : children) {
+                Component found = getChildNamed(child, name);
+                if (found != null) {
+                    return found;
+                }
+            }
+        }
+        return null;
+    }
+}
\ No newline at end of file
